site/content/docs/01-component-format.md

```diff
@@ -57,6 +57,8 @@ In development mode (see the [compiler options](docs#svelte_compile)), a warning
 
 If you export a `const`, `class` or `function`, it is readonly from outside the component. Function *expressions* are valid props, however.
 
+Readonly props can be accessed as properties on the element, tied to the component using [`bind:this` syntax](docs#bind_element).
```

Comment:
i believe this is not limited to readonly props, but rather all props right?

Reply:
Theoretically yes, but only when setting `accessors` to true on the component. If you try to access non-readonly props otherwise, you will get:
```
Props cannot be read directly from the component instance unless compiling with 'accessors: true' or '<svelte:options accessors/>'
```

readonly props should be accessible regardless from the component instance.

Here is a REPL showing this behavior: 
https://svelte.dev/repl/62235b43f41948d4bd2b03221668a246?version=3.31.0